### ISSUE TITLE
`use-test`: Allow import statements without default specifier

### ISSUE THREAD

**Issue by exodustx0** [avajs/eslint-plugin-ava#320]
The `use-test` rule is triggered even if the import statement does not include a default specifier:

```ts
import type { TestInterface } from 'ava'; // "AVA should be imported as `test`."
```

Not a high priority issue I don't think, but it should be a quick fix, and ensure that `use-test` only concerns itself with `test`, not anything else.

**Comment by sindresorhus** [comment]
If anyone wants to work on this, see the initial attempt in https://github.com/avajs/eslint-plugin-ava/pull/321.
